lib/ui/creators_corner_page.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class CreatorsCornerPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    // TODO: WP-13 (ubclaunchpad.atlassian.net/browse/WP-13)
+    return Center(
+      child: Container(
+        child: const Text(
+          "Creator's Corner Page",
+          style: TextStyle(fontSize: 30),
```

Comment:
Might be good to have the styling details as constants in a separate file for reusability & ease of change!